### PR TITLE
Watersmart 496

### PR DESCRIPTION
**Pull request by mamcderm-usgs**
- completed tests for watersmart-496
- some cleanup made while working on tests

### CODE REVIEW

File: nwc/src/main/webapp/js/view/WaterBudgetHucDataView.js [hunk 1]
```diff
@@ -156,7 +155,6 @@ NWC.view.WaterBudgetHucDataView = NWC.view.BaseView.extend({
 	},
 
 	displayCountyMap : function() {
-		$('#county-selection-div').show();
```
Comment (mbucknell) [inline nwc/src/main/webapp/js/view/WaterBudgetHucDataView.js]: Don't we still need this line?

**Comment by mbucknell** [comment]
:+1: , except for the comment above.

File: nwc/src/main/webapp/js/view/WaterBudgetHucDataView.js (hunk 1 above)
Comment (mamcderm-usgs) [inline nwc/src/main/webapp/js/view/WaterBudgetHucDataView.js]: I believe this is superfluous, unless I'm missing something.  It's just an empty div until you click the "show counties" button.  I'm going to go ahead and merge but I wouldn't rule out needing to put this back at some point.

Comment (mbucknell) [inline nwc/src/main/webapp/js/view/WaterBudgetHucDataView.js]: Ok ... You are correct. Merge away!

**Pull request merged**